app/src/main/java/org/groundplatform/android/persistence/remote/firebase/FirestoreDataStore.kt

```diff
@@ -71,6 +71,10 @@ internal constructor(
     )
   }
 
+  override fun getPublicSurveyList(): Flow<List<SurveyListItem>> = flow {
+    emitAll(db().surveys().getPublicReadable().map { list -> list.map { it.toListItem(false) } })
```

Comment:
Use named arg for `toListItem()` boolean param to make more self descriptive. (`argName = false`)